src/attributes.jl

```diff
@@ -337,6 +337,27 @@ An optimizer attribute for the string identifying the solver/optimizer.
 """
 struct SolverName <: AbstractOptimizerAttribute end
 
+"""
+    Silent
+
+An optimizer attribute for silencing the output of an optimizer. When `set`
+to `true`, it takes precedence over any other attribute controlling verbosity
+and requires the solver to produce no output. The default value is `false`
+which has no effect, the verbosity is controlled by other attributes.
```

Comment:
Run-on sentence:
The default value is `false` which has no effect. In this case the verbosity is controlled by other attributes.